pkg/transform/testdata/expected-report-oauth.json

```diff
@@ -67,6 +67,13 @@
           "confidence": 2,
           "comment": "Identity provider my_openid_connect is supported in OCP4"
         },
+        {
+          "name": "Issuer",
+          "kind": "IdentityProviders:OpenIDIdentityProvider",
+          "supported": true,
+          "confidence": 2,
+          "comment": "OCP4 requires an 'issuer' URL, please edit OAuth manifest file and configure this field"
```

Comment:
@gildub do we have other places in the code where we require an action from user before the generated CRs could be used? I wonder, whether we want to stress even more that there are actions that are required to be fulfilled. WDYT?

The code looks Okay-ish to me though.

Reply:
@arapov, we can survey user at any time but do we really want to introduce such interaction during execution of the tool?  I suppose that's a choice to make.

@jwmatthews what do you think?

Reply:
I would refrain from survey at this point in time, OK to require the user to review and modify the CR before being applied.

I think the note in the report that 'issuer' needs to be set is helpful and sufficient.

It's OK to have the CR have an empty field for the issuer and have it fail to be applied as described in
https://bugzilla.redhat.com/show_bug.cgi?id=1755865

I assume using 'https://no-issuer.me" could be confusing and hide the error, so would remove 'issuer: https://no-issuer.me' and go back to issuer: ""

Reply:
@jwmatthews, thanks for your input, and helping us make a decision.

Let's reverse the default value to "" and keep the report message then.

Reply:
+1 for the message and failing on import to ocp4.

Reply:
@jwmatthews, thanks for your input, and helping us make a decision.

Let's reverse the default value to "" and keep the report message then.